src/index.html

```diff
@@ -75,377 +76,349 @@
         </ul>
       </nav>
     </header>
-
-    <main class="main">
-      <div class="catalog">
-        <div class="card catalog__card" data-qa="card">
+    <main class="catalog">
+      <div class="catalog__wrapper">
+        <article
+          class="card catalog__card"
+          data-qa="card"
+        >
+        
           <img
             src="images/imac.jpeg"
             alt="APPLE A1419 iMac 27"
-            class="card__picture"
+            class="card__product-image"
           >
 
-          <div class="card__name">
-            <h2 class="card__name-title">
-              APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
-            </h2>
-          </div>
+          <h2 class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
```

Comment:
add `&quot;`  instead double quotes (")
(My bad, in last reviev I wrote uncorrect message)